ddtrace/contrib/internal/langgraph/patch.py

```diff
@@ -98,6 +98,61 @@ async def traced_runnable_seq_ainvoke(langgraph, pin, func, instance, args, kwar
     return result
 
 
+@with_traced_module
+def traced_runnable_seq_astream(langgraph, pin, func, instance, args, kwargs):
+    """
+    Typically, RunnableSeq.ainvoke() is called when being run as a task for its parent Pregel (CompiledGraph).
+    However, when using Pregel.astream_events() (inherited from langchain's Runnable), it calls RunnableSeq.astream()
+    instead.
+
+    This function returns a generator wrapper that yields the results of RunnableSeq.astream(),
+    ending the span after the stream is consumed, otherwise following the logic of traced_runnable_seq_ainvoke().
+    """
+    integration: LangGraphIntegration = langgraph._datadog_integration
+
+    node_name = _get_node_name(instance)
+
+    if node_name in ("_write", "_route"):
+        return func(*args, **kwargs)
+    if node_name == "LangGraph":
+        config = get_argument_value(args, kwargs, 1, "config", optional=True) or {}
+        config.get("metadata", {})["_dd.subgraph"] = True
+        return func(*args, **kwargs)
+
+    span = integration.trace(
+        pin,
+        "%s.%s.%s" % (instance.__module__, instance.__class__.__name__, node_name),
+        submit_to_llmobs=True,
+    )
+    result = None
+    try:
+        result = func(*args, **kwargs)
+    except Exception:
+        span.set_exc_info(*sys.exc_info())
+        integration.llmobs_set_tags(span, args=args, kwargs=kwargs, response=None, operation="node")
+        span.finish()
+        raise
+
+    async def _astream():
+        item = None
+        while True:
+            try:
+                item = await result.__anext__()
+                yield item
+            except StopAsyncIteration:
+                response = item[-1] if isinstance(item, tuple) else item
```

Comment:
this needs to be fixed - the `astream` function for `RunnableSeq` actually does not yield anything, it just sends the output to the run manager for the `on_chain_end` event ([ref](https://github.com/langchain-ai/langgraph/blob/2b8c295898bd35f5d74f096cff70c1c7df93f124/libs/langgraph/langgraph/utils/runnable.py#L809-L815)). exploring some workarounds so we can annotate the output properly for LLMObs spans